.github/workflows/rust.yml

```diff
@@ -23,7 +24,8 @@ jobs:
           - stable
           - beta
           - nightly
-          - 1.51.0
+          # RedisJSON literally will refuse to compile in this version of rust
```

Comment:
We don't want to remove Rust 1.51. There's no reason for RedisJSON to be built using the same Rust versions that the client library uses -- `stable` should be fine.

Reply:
you make a very good point, I'll add a `+stable` to the cargo build command for building redisjson